model/host/host.go

```diff
@@ -205,6 +206,43 @@ type DockerOptions struct {
 	EnvironmentVars []string `mapstructure:"environment_vars" bson:"environment_vars,omitempty" json:"environment_vars,omitempty"`
 }
 
+func (opts *DockerOptions) FromDistroSettings(d distro.Distro, _ string) error {
+	if len(d.ProviderSettingsList) != 0 {
```

Comment:
are we ever going to end up in case where we hit this case on a host's copy of the distro options and we'll have turned it into one item list and then we use this.

Reply:
Yes, but that's what I want, since at some point we're getting rid of ProviderSettings and just using ProviderSettingsList.